wiki/Mascots/es.md

```diff
@@ -10,31 +6,33 @@ Un vídeo de YouTube mostrando las mascotas de osu! puede verse en [Mascot Showc
 
 ## Oficiales
 
-### pippi
+### ![osu! icon](/wiki/shared/mode/osu.png) pippi
```

Comment:
the alt-text of the image should be translated (same for images below)

for ref:

`![alt-text](link)`